routing/localchans/manager.go

```diff
@@ -311,8 +311,16 @@ func (r *Manager) createEdge(channel *channeldb.OpenChannel,
 			err)
 	}
 
+	// We need to make sure we use the real scid for public confirmed
+	// zero-conf channels.
+	shortChanID := channel.ShortChanID()
+	isPublic := channel.ChannelFlags&lnwire.FFAnnounceChannel != 0
+	if isPublic && channel.IsZeroConf() && channel.ZeroConfConfirmed() {
+		shortChanID = channel.ZeroConfRealScid()
+	}
+
 	info := &models.ChannelEdgeInfo{
-		ChannelID:    channel.ShortChanID().ToUint64(),
+		ChannelID:    shortChanID.ToUint64(),
```

Comment:
don't we have itest to check this behavior? Or can you update one of the zero conf itests to show the before/after effect?

Reply:
also update the `ChannelEdgePolicy` below?